src/editor/UI/IndentLineRenderer.re

```diff
@@ -60,51 +93,69 @@ let render =
     };
 
     incr(l);
+    previousIndentLevel := level;
+
+    if (line == cursorLine) {
+      cursorLineIndentLevel := level;
```

Comment:
Not actually needed for this....was just since we already calculate this, seemed pointless to do it again a few lines later.